config.py

```diff
@@ -128,7 +128,7 @@ class Development(Config):
     DM_DOCUMENTS_BUCKET = "digitalmarketplace-dev-uploads"
     DM_ASSETS_URL = "https://{}.s3-eu-west-1.amazonaws.com".format(DM_SUBMISSIONS_BUCKET)
 
-    DM_NOTIFY_API_KEY = "not_a_real_key-00000000-fake-uuid-0000-000000000000"
+    DM_NOTIFY_API_KEY = "katsandbox-95316ff0-e555-462d-a6e7-95d26fbfd091-83356680-310b-4f0f-a25d-613fb10066de"
```

Comment:
I think this is better as an obviously fake key, so we don't end up using your sandbox key?

Reply:
I've revoked that key anyway.